.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "doubtfire-web"]
 	path = doubtfire-web
-	url = https://github.com/doubtfire-lms/doubtfire-web.git
+	url = git@github.com:thoth-tech/doubtfire-web.git
```

Comment:
**suggestion:** I am curious of why we change to the `ssh` protocol? `git` protocol requires the users to set up ssh keys and to my knowledge, Windows users might find it is a bit hard to configure.

Reply:
Yes you have add a public ssh key to github account.